Jenkinsfile-deploy

```diff
@@ -0,0 +1,35 @@
+pipeline {
+    agent {
+        label "master"
+    }
+    parameters {
+        choice(name: "STAGE", choices: ["intg", "staging", "prod"], description: "The stage you are building the front end for")
```

Comment:
Probably want to change the description in the parameters section

Reply:
Yeah good point. I've changed the other file as well.